core/src/test/java/suites/BuildVerificationTestSuite.java

```diff
@@ -38,7 +37,6 @@
         QueueRepositoryTester.class,
         ApiAuthenticationTests.class,
         SystemPropTests.class,
-        ParallelWorkerTests.class,
```

Comment:
yes see the ExcludeCategories above? as long as test marked there have one of those they will be excluded